app/i18n/nl/gen.php

```diff
@@ -73,7 +73,10 @@
 		'Jun' => '\\J\\u\\n\\i',
 		'jun' => 'jun',
 		'june' => 'Jun',
+		'last_2_year' => 'Last two years',	// TODO - Translation
```

Comment:
@Frenzie Maybe you could suggest the translation?

Reply:
```suggestion
		'last_2_year' => 'Afgelopen twee jaar',
```

Reply:
```suggestion
		'last_2_year' => 'Afgelopen twee jaar',
```